Mochi Diffusion/Views/JobQueueView.swift

```diff
@@ -143,19 +143,19 @@ private struct InfoPopoverView: View {
 
             controller.currentModel = config.model
 
-            if let startingImage = config.pipelineConfig.startingImage {
+            if let startingImage = config.pipelineConfig.initImage {
```

Comment:
Even if `strength` is guaranteed to be non-nil when `initImage` is non-nil (is it?) it is safer and clearer to safely unwrap them both here instead of force unwrapping strength with !
```suggestion
            if let startingImage = config.pipelineConfig.initImage, let strength = config.pipelineConfig.strength {
```

and again on line 223